test/cctest/test_util.cc

```diff
@@ -90,10 +90,6 @@ TEST(UtilTest, ToLower) {
   EXPECT_EQ('a', ToLower('A'));
 }
 
-namespace node {
-  void LowMemoryNotification() {}
-}
-
```

Comment:
Yeah, it was intentional and just like you said it conflicts with the function from util.cc. Any better way around that you can think of?